lib/subghz/protocols/bett.c

```diff
@@ -242,7 +242,6 @@ void subghz_protocol_decoder_bett_feed(void* context, bool level, uint32_t durat
         if(!level) {
             if(DURATION_DIFF(duration, subghz_protocol_bett_const.te_short * 44) <
                (subghz_protocol_bett_const.te_delta * 15)) {
-                instance->decoder.parser_step = BETTDecoderStepSaveDuration;
```

Comment:
bring it back. this is magic, you need the decoder to continue decoding the stream without looking for a new guard time

Reply:
:magic_wand: